PLUB/Sources/Views/Meeting/MeetingViewController.swift

```diff
@@ -170,7 +170,7 @@ extension MeetingViewController: UICollectionViewDelegate, UICollectionViewDataS
   func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
     if indexPath.row < meetingList.count {
       // 플러빙 메인
-      let vc = MainPageViewController()
+      let vc = MainPageViewController(plubbingID: 0)
```

Comment:
MeetingViewController에서는 이미 Plub 정보를 가지고 처리하고 있습니다. 아래와 같이 `plubbingID`를 넘겨주시면 됩니다.

```suggestion
      let vc = MainPageViewController(plubbingID: meetingList[indexPath.row].plubbingID)
```

Reply:
넵 수정완료하였습니다, 또한 저 부분외에 헤더 클릭시에 ClipboardViewController로 푸시될때 plubbingID도 넘기는 코드 추가하였습니다 👍👍